src/backend/joanie/core/serializers.py

```diff
@@ -245,24 +244,6 @@ class Meta:
             "products",
         ]
 
-    def get_orders(self, instance):
```

Comment:
I think you have also to remove the `OrderLiteSerializer` which is now unused if I remember well.